pkg/cloud/assets/openstack/deployment.yaml

```diff
@@ -0,0 +1,64 @@
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: openstack-cloud-controller-manager
+  namespace: openshift-cloud-controller-manager
+  labels:
+    app: openstack-cloud-controller-manager
+spec:
+  selector:
+    matchLabels:
+      app: openstack-cloud-controller-manager
+  template:
+    metadata:
+      labels:
+        app: openstack-cloud-controller-manager
+    spec:
+      hostNetwork: true
+      serviceAccount: cluster-cloud-controller-manager
+      priorityClassName: system-cluster-critical
+      nodeSelector:
+        node-role.kubernetes.io/master: ""
+      tolerations:
+        - key: CriticalAddonsOnly
+          operator: Exists
+        - key: node-role.kubernetes.io/master
+          operator: Exists
+          effect: "NoSchedule"
+        - key: node.cloudprovider.kubernetes.io/uninitialized
+          value: "true"
+          effect: NoSchedule
+      containers:
+        - name: openstack-cloud-controller-manager
+          image: docker.io/k8scloudprovider/openstack-cloud-controller-manager:latest
```

Comment:
This should be managed by `CCCMO` and match the image version from in-cluster image registry., out-of-scope for this PR though.